sdk/storage/Azure.Storage.DataMovement/src/StorageTransferProgress.cs

```diff
@@ -7,27 +7,27 @@
 namespace Azure.Storage.DataMovement
 {
     /// <summary>
-    /// Progress Handler to track how many files and bytes were transferred, along with files that failed or were skipped in transfer
+    /// Progress Handler to track how many files and bytes were transferred, along with files that failed or were skipped in transfer.
     /// </summary>
     internal class StorageTransferProgress
     {
         /// <summary>
-        /// Number of Files that succeeded in transfer
+        /// Number of files that were transferred successfully.
         /// </summary>
         public int CompletedCount { get; internal set; }
 
         /// <summary>
-        /// Number of files that were skipped in transfer due to overwrite being set to not overwrite files, but the files exists already
+        /// Number of files that were skipped in transfer due to files already existing in the destination and overwite policy set to not overwrite existing files.
```

Comment:
Maybe we should rewrite this to be

Number of files that were skipped in transfer due to files already existing in the destination. Files will be skipped, if the overwrite policy is set to not overwrite existing files.